src/Client.php

```diff
@@ -16,15 +16,21 @@ class Client extends ElasticaClient
     /* Elastically config keys */
     const CONFIG_MAPPINGS_DIRECTORY = 'elastically_mappings_directory';
     const CONFIG_INDEX_CLASS_MAPPING = 'elastically_index_class_mapping';
+    const CONFIG_INDEX_PREFIX = 'elastically_index_prefix';
     const CONFIG_SERIALIZER_CONTEXT_PER_CLASS = 'elastically_serializer_context_per_class';
     const CONFIG_SERIALIZER = 'elastically_serializer';
     const CONFIG_BULK_SIZE = 'elastically_bulk_size';
 
     private $indexer;
+    private $indexBuilder;
 
     public function getIndexBuilder(): IndexBuilder
     {
-        return new IndexBuilder($this, $this->getConfig(self::CONFIG_MAPPINGS_DIRECTORY));
+        if (!$this->indexBuilder) {
+            $this->indexBuilder = new IndexBuilder($this, $this->getConfig(self::CONFIG_MAPPINGS_DIRECTORY));
```

Comment:
This not really DI friendly :/

Reply:
What do you recommand?

The IndexBuiler needs the Client, the Client needs the IndexBuilder :grimacing:

Reply:
Oh, so maybe it's a design issue. Let's keep it for now, we will see how to make it better later